packages/gatsby/static/configuration/manifest.json

```
@@ -330,6 +330,11 @@
           "type": "string",
           "enum": ["workspaces", "dependencies", "none"],
           "default": "none"
+        },
+        "selfReferences": {
+          "description": "Defines whether workspaces are allowed to require themselves - results in creation of self-referencing symlinks, overriding for the current workspace the value initially set for [`nmSelfReferences`](/configuration/yarnrc#nmSelfReferences)",
+          "type": "boolean",
+          "default": true
         }
       },
       "_exampleKeys": ["hoistingLimits"],
```

The example won't show up in the docs unless you add an entry to `_exampleKeys`:

```suggestion
      "_exampleKeys": ["hoistingLimits", "selfReferences"],
```

---

Thank you Paul!